api4/channel.go

```diff
@@ -421,6 +422,67 @@ func updateChannelMemberRoles(c *Context, w http.ResponseWriter, r *http.Request
 	ReturnStatusOK(w)
 }
 
+func addChannelMember(c *Context, w http.ResponseWriter, r *http.Request) {
+	c.RequireChannelId()
+	if c.Err != nil {
+		return
+	}
+
+	data := model.MapFromJson(r.Body)
+	if data == nil {
+		c.SetInvalidParam("channel_member")
+		return
+	}
+
+	userId := data["user_id"]
+	if len(userId) != 26 {
+		c.SetInvalidParam("user_id")
+		return
+	}
```

Comment:
Can we actually parse the request into a ChannelMember struct? Might change the request body a bit and then just need to set ` member.ChannelId = c.Params.ChannelId` to make sure we add them to the channel provided in the URL